src/test/resources/ssl/client.crt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please replace certificates with openssl certificate generation. These things are typically subject to expire and then no one knows how to set them up properly because certificates and keys aren't the most readable things. 

Take a look at Spring Vault's [`create_certificates.sh`](https://github.com/spring-projects/spring-vault/blob/master/src/test/bash/create_certificates.sh) that creates certificates as part of its build. We assume openssl availability but that should be less of an issue.